### PR TITLE
Nav feature

### PR DESCRIPTION
**Pull request by zainsadaqat**
### In this Pull Request, we implemented the following:

- A Navigation bar.
- Three content sections:
  - Books list.
  - Add book form.
  - Contact info.
- Date

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,14 +13,44 @@
...
+            class="nav-link active"
+            aria-current="page"
+            href="index.html"
+            >Add New</a
+          >
+        </li>
+        <li class="nav-item">
+          <a id="contact-view" class="nav-link" href="#">Contact</a>
+        </li>
+      </ul>
```
Comment (rloterh) [inline index.html]: - [x] The right menu elements in your navbar is missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.
- [x] To enhance the user experience of this design kindly ensure that an active menu link is highlighted (should have color change or change in opacity  of the active menu item). In other words, the current active page should have the corresponding menu item highlighted.

**Your design menu:**
image

**The wireframe menu design:**
image

File: index.html [hunk 2]
```diff
@@ -63,6 +93,76 @@ <h2 class="text-primary">Add a new Book</h2>
...
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/zainsadaqat"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+      </ul>
```
Comment (rloterh) [inline index.html]: -  [x] The contents of your contact page does match the provided wireframe design template. As we want our design to as much as possible closely follow the the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) kindly ensure to:
- **remove borders** around the `Our email`, ` Our phone number` and `Our address` elements
- Provide a **realistic**  _email_, _contact_ and _address_ for `Our email`, ` Our phone number` and `Our address`
- align `Our email`, ` Our phone number` and `Our address` to the **left**, and **bulleted** as can be observed in the template design

**Your design:**
!image

**The wireframe design:**
image

File: index.html [hunk 3]
```diff
@@ -63,6 +93,76 @@ <h2 class="text-primary">Add a new Book</h2>
...
+    </div>
```
Comment (rloterh) [inline index.html]: - [x] Your design pages does not include a `footer` as can be seen in the [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) design. Kindly ensure to add the `footer` section to each page of your design.

**The wireframe template (footer section):**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Team, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚆 🚥 

## Good Points 👍  
- App is working as intended
- Correct usage of _luxon.js_
- No linter errors


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _Kindly check the comments marked [OPTIONAL] under the review._

You can also consider:
- [OPTIONAL] 
Kindly endeavor to update the `Demo link` in the ReadMe to reflect the current project. 


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi  @DanSam5K and @zainsadaqat , 

Great work with making all the changes requested by the previous TSE :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DanSam5K** [comment]
## Hello @sinansevgi 😊 
- Thank you for your kind feedback 🤝 
💪

**Pull request merged**